lib/ofex.ex

```diff
@@ -47,14 +47,21 @@ defmodule Ofex do
   def parse(data) do
     case validate_ofx_data(data) do
       {:ok, parsed_ofx} ->
-        parsed_ofx
+        result = parsed_ofx
         |> xpath(~x"//OFX/*[contains(name(),'MSGSRS')]"l)
         |> Enum.map(&parse_message_set(xpath(&1, ~x"name()"s), &1))
-        |> Map.new
+        |> List.flatten
+        |> Enum.reduce(%{signon: %{}, accounts: []}, &reduce_items_list/2)
```

Comment:
Since we're assigning this now, can we space all the `|>` piped commands under `parsed_ofx`